src/main/java/taxi/model/Driver.java

```diff
@@ -50,11 +75,13 @@ public boolean equals(Object o) {
         Driver driver = (Driver) o;
         return Objects.equals(id, driver.id)
                 && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
+                && Objects.equals(licenseNumber, driver.licenseNumber)
+                && Objects.equals(login, driver.login)
+                && Objects.equals(password, driver.password);
```

Comment:
I would not recommend to add password to equals/hashcode methods.
Further it will be encrypted and depending on hashing algorithm this may cause improper comparison